TestScriptWebPage/src/InstallationPage.css

```diff
@@ -58,198 +59,150 @@ body {
-.test-card.changed { border-left-color: var(--yellow); }
 
-.test-card__head {
+/* INPUTS — Panel Base + Sky Blue Focus */
+.inputs-grid {
   display: flex;
-  align-items: center;
-  gap: 10px;
-  padding: 12px 14px;
-  border-bottom: 1px solid var(--border);
-}
-.test-badge {
-  text-transform: uppercase;
-  font-size: 11px;
-  font-weight: 800;
-  padding: 4px 8px;
-  border-radius: 999px;
-  letter-spacing: .6px;
-  color: #0e0e0e;
-  background: #999;
-}
-.test-card.pass .test-badge    { background: #2fd37a; }
-.test-card.fail .test-badge    { background: var(--orange); }
-.test-card.pending .test-badge { background: var(--yellow); }
-.test-card.changed .test-badge { background: var(--yellow); }
-.test-title { margin: 0; font-size: 14px; color: var(--text); }
-
-/* code block */
-pre.code {
-  margin: 0;
-  padding: 12px 14px;
-  font-family: ui-monospace, SFMono-Regular, Menlo, monospace;
-  font-size: 12.5px;
-  line-height: 1.5;
-  background: linear-gradient(180deg, rgba(255,255,255,0.02), rgba(0,0,0,0.06));
+  flex-direction: column;
+  flex-wrap: wrap;
+  font-size: medium;
+  font-weight: 200%;
 }
-.code-line {
+.input-block {
   display: flex;
-  gap: 12px;
-  padding: 2px 0;
+  font-weight: 600;
+  flex-direction: column;
 }
-.code-line .ln {
-  width: 32px;
-  text-align: right;
-  color: var(--muted);
-  user-select: none;
-  opacity: .65;
-}
-
-/* line-level highlight */
-.test-card.fail .code-line.line-attn {
-  background: rgba(251,133,0,0.16);
-  outline: 1px dashed rgba(251,133,0,0.55);
+.input-label {
+  color: rgb(255, 255, 255);
+  font-size: 1.7rem;
 }
-.test-card.pending .code-line.line-attn,
-.test-card.changed .code-line.line-attn {
-  background: rgba(255,183,3,0.14);
-  outline: 1px dashed rgba(255,183,3,0.45);
+.long-input {
+  width: fit-content;
+  font-size: 1.5rem;
+  resize: vertical;
+  padding: 12px 14px;
+  border-radius: 10px;
+  color: rgba(255, 255, 255, 0.837);
+  background: rgba(0, 0, 0, 0.675);
 }
 
-/* CTA (optional, unchanged from prior) */
-button.cta {
-  background: var(--yellow);
-  color: var(--prussian-blue);
-  padding: 10px 20px;
-  border: none;
-  border-radius: 8px;
-  font-weight: bold;
-  cursor: pointer;
-  transition: background 0.2s ease;
-}
-button.cta:hover {
-  background: var(--orange);
-  color: white;
+.testscript-installation {
+  display: flex;
+  justify-content: center;
+  margin: 5rem 0 22px;
+  font-size: 3rem;
+  color: rgb(255, 255, 255);
+  text-shadow: 2px 2px 0 rgba(0, 0, 0, 0.778), -2px -2px 0 rgba(0, 0, 0, 0.778),
+    2px -2px 0 rgba(0, 0, 0, 0.778), -2px 2px 0 rgba(0, 0, 0, 0.778),
+    0px 2px 0 rgba(0, 0, 0, 0.778), 2px 0px 0 rgba(0, 0, 0, 0.778),
+    0px -2px 0 rgba(0, 0, 0, 0.778), -2px 0px 0 rgba(0, 0, 0, 0.778);
+}
+.demo-video {
+  position: relative;
+  width: 100%;
+  max-width: 800px;
+  aspect-ratio: 16 / 9;
+  margin: 4rem auto;
+  box-shadow: 0 10px 30px rgba(0, 0, 0, 0.25);
+  border-radius: 12px;
```

Comment:
Using `px` will make a page non-responsive on different sizes. Can we convert to `em`?